internal/pkg/api/server/templates/service/mapper.go

```diff
@@ -183,7 +184,6 @@ func ComponentsResponse(d dependencies.ForProjectRequest, in []string) (out []st
 		}
 		out = append(out, componentId)
 	}
-	sort.Strings(out)
```

Comment:
Podla poradia `components` sa v UI vykresluju ikonky, takze to nemozeme zoradrovat.